api/who/who.proto

```diff
@@ -21,6 +31,21 @@ message PutLocationRequest {
   double longitude = 3;
 }
 
+message GetGlobalStatsResponse {
```

Comment:
Are these intended to be global stats? Guess we can add fields for local stats later?

Reply:
Yes global only.